frontendUiTable/src/main/java/com/virtuslab/gitmachete/frontend/ui/table/GitMacheteGraphTable.java

```diff
@@ -111,7 +121,7 @@ public void setTextForEmptyGraph(String upperText, String lowerText) {
 
   @Override
   @UIEffect
-  public void updateUI(@UnknownInitialization GitMacheteGraphTable this) {
```

Comment:
Hmm I think we should still be aware of this method being possibly called from `JTable`'s c'tor, see comments to astubs